src/persistence/profile.cpp

```diff
@@ -340,6 +396,12 @@ void Profile::saveToxSave(QByteArray data)
     }
 }
 
+/**
+@brief Gets the path of the avatar file cached by this profile and corresponding to this owner ID.
+@param ownerId Path to avatar of friend with this ID will returned.
+@param forceUnencrypted If forceUnencrypted, we return the path to the plaintext file even if we're an encrypted profile.
```

Comment:
If true, return the path to the plaintext file even if this is an encrypted profile.